src/main/java/com/fasterxml/jackson/dataformat/xml/ser/XmlSerializerProvider.java

```diff
@@ -56,6 +55,11 @@ public XmlSerializerProvider(XmlSerializerProvider src,
      */
 
     @Override
+    public DefaultSerializerProvider copy() {
+        return this;
```

Comment:
A brief comment would be appreciated... however, are you sure this is safe? Keep in mind that it extends `DefaultSerializerProvider` so I think it would be better to create a copy (super-class single-arg constructor to use).